modules/PrimaryPool/substreams.yaml

```diff
@@ -27,13 +34,27 @@ modules:
     kind: map
     initialBlock: 8586768
     inputs:
+      - source: sf.ethereum.type.v2.Block
+      # Added UP
       - map: map_pools
```

Comment:
@yaksheshgupta are we using map_subscriptions or map_subscriptionscheck ? One seems to be taking map pools as input - is this what you were mentioning yesterday on telegram ?